File: P2Ejercicio3.c
#include<stdio.h>

void main(void){

int a[3];
int b=0;
int c=0;
int res=0;

printf("Bienvenido,ingrese 3 números, el módulo y la operación que desea realizar. \n");
printf("Ingrese la operación que desea realizar: \n1.Suma. \n2.Resta. \n3.Multiplicación. \n");
scanf("%d",&c);

printf("Escriba el primer número: ");
scanf("%d",&a[0]);
printf("Escriba el segundo número: ");
scanf("%d",&a[1]);
printf("Escriba el tercer número: ");
scanf("%d",&a[2]);
printf("Escriba el módulo: ");
scanf("%d",&b);

if(c<4 && c>0){
	switch(c){
		case 1:
		res=(a[0]%b)+(a[1]%b)+(a[2]%b);
		printf("Resultado=%d \n",res);
		break;

                case 2:
                res=(a[0]%b)-(a[1]%b)-(a[2]%b);
                printf("Resultado=%d \n",res);
                break;

                case 3:
                res=(a[0]%b)*(a[1]%b)*(a[2]%b);
                printf("Resultado=%d \n",res);
                break;

	}
}
else{
printf("El número ingresado no es válido");
}
}

File: P2Ejercicio3.py
def modulo():
	print("Bienvenido, a continuacion se le pedirán 3 valores, el módulo y la operación que desea realizar para que el resultado mostrado sea la operación elegida de las 3 operaciones.")
	e=int(input("Ingrese el número de la operación que desea realizar: \n1.Suma. \n2.Resta. \n3.Multiplicación. \n" ))
	if(0<e<4):
		a=int(input("Ingrese el primer valor: "))
		b=int(input("Ingrese el segundo valor: "))
		c=int(input("Ingrese el tercer valor: "))
		d=int(input("Ingrese el módulo: "))
		if(e==1):
			res=(a%d)+(b%d)+(c%d)
			print("Resultado="+str(res))
		elif(e==2):
			res=(a%d)-(b%d)-(c%d)
			print("Resultado="+str(res))
		elif(e==3):
			res=(a%d)*(b%d)*(c%d)
			print("Resultado="+str(res))
	else:
		print("El valor ingresado no es válido.")
pass
modulo()
